lib/modules/apostrophe-assets/index.js

```diff
@@ -1288,7 +1306,10 @@ module.exports = {
         // this property won't be set
         middleware.push(self.lessMiddleware);
       }
-      middleware.push(self.apos.express.static(self.apos.rootDir + '/public'));
+      middleware.push(self.apos.express.static(
+        self.apos.rootDir + '/public',
+        _.defaults({}, self.options.serveStaticOptions)
```

Comment:
Does `_.defaults` guarantee reasonable behavior if `self.options.serveStaticOptions` is null? I don't see it in the docs. I would usually use `|| {}` here.

Also, it's a given that things in `options` are options, so we should just call it `static` and in context it's clear that it contains options.

I suggest we just say `static` because the express documentation, which is what people are probably looking at, never explicitly says express.static is serve-static, although that is what is in the source. And brevity is nice if there's no lack of clarity.